wiki/Chat_Console/en.md

```diff
@@ -1,71 +1,69 @@
+| Colour | Who?                                                                                                                                                                                                     |
+|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **White**  | You                                                                                                                                                                                                      |
+| **Pale**   | Non-supporters                                                                                                                                                                                           |
+| **Yellow** | ![Supporter Tag](/wiki/shared/osu!supporter.png "Supporter Tag") Supporters                                                                                                                                            |
+| **Red**    | [Global Moderation Team](/wiki/People/Global_Moderation_Team/ "Global Moderation Team") or [Quality Assurance Team](/wiki/People/Quality_Assurance_Team/ "Quality Assurance Team") member      |
+| **Green**  | Line contains your name or certain keywords that you have added to trigger a "highlight". A copy of the message will also appear in a dedicated channel, \#highlight, which contains all of these lines. |
+| **Blue**   | Private Message                                                                                                                                                                                          |
+| **Cyan**   | [peppy](https://osu.ppy.sh/u/2 "peppy"), the creator of *osu!*                                                                                                                                            |
+| **Pink**   | [BanchoBot](/wiki/BanchoBot/ "BanchoBot")                                                                                                                                                           |
+
+-   Click the `Show Ticker` box to display the newest chat message at the bottom of the screen when the chat console is not visible.
+-   Click the `Auto-Hide` box to automatically hide the chat during gameplay (excluding beatmap intro, outro and breaks).
+-   Click the `Hide Chat` box or press F8 again to hide the chat console.
 
 Extended Chat Console
 =====================
 
-*[osu!academy](osu!academy "wikilink") covered this page in [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) along with [Multiplayer](Multiplayer "wikilink").*
+*[osu!academy](/wiki/Announcements/osu!academy/ "osu!academy") covered this page in [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) along with [Multiplayer](/wiki/Multi/ "Multiplayer").*
```

Comment:
Just need to change to https:// from http://
Also, if it's not too much to ask(it probably is), I think you need to apply the changes to the other languages as well.